### PR TITLE
Feature/command structure v2

### PR DESCRIPTION
**Pull request by JohannesRudolph**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,3 @@
-.vscode
-
```
Comment (tfelix) [inline .gitignore:2]: t: A quick search showed my that its not super uncommen to have this shared, but keep in mind that this might be dangerous if some secrets are placed in those files.

File: src/import_map.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "imports": {
+    "/": "./",
+    "./": "./",
```
Comment (tfelix) [inline src/import_map.json:4]: d: why is this required? 
Also is `std/fmt/` even used? (I closed some files already but can not see it in the still open files, so I am wondering if this has any effect at all).

**Review by tfelix — APPROVED** [review]
lgtm

File: src/import_map.json (hunk 2 above)
Comment (JohannesRudolph) [reply src/import_map.json:4]: I stole that trick from the deno manual, it allows absolute import paths https://deno.land/manual/linking_to_external_code/import_maps

I really like those over relative import paths, they make moving code around much easier (e.g. if I move a file from `foo/bar` up one level to `foo/`, I don't have to fix all my relative import paths from `../../` to `../`
